clang/lib/Headers/avx512dqintrin.h

```diff
@@ -167,6 +167,8 @@ _mm512_maskz_mullo_epi64(__mmask8 __U, __m512i __A, __m512i __B) {
                                              (__v8di)_mm512_setzero_si512());
 }
 
+#pragma float_control(push)
```

Comment:
Needs tests

Reply:
Thanks, this is just an experiment. Unfortunately it doesn't help with https://github.com/llvm/llvm-project/issues/118152, so I turn it into draft. Will be back to it if I find useful scenarios.